### PR TITLE
Enhance code issues

### PR DESCRIPTION
**Pull request by martinjaimem**
***Nice Repo!!!***

I did a small enhancement of the client class, removed an unused variable, and removed an empty line after the class name (which in the rest of the code it is not being added).

Also, I'd like to suggest adding some kind of linter to the code like Rubocop WDYT?

Thanks!!!

### CODE REVIEW

File: lib/harvesting/client.rb [hunk 1]
```diff
@@ -21,16 +20,18 @@ def initialize(access_token: ENV['HARVEST_ACCESS_TOKEN'], account_id: ENV['HARVE
       @access_token = access_token.to_s
       @account_id = account_id.to_s
 
-      if @account_id.length == 0 || @access_token.length == 0
-        raise ArgumentError.new("Access token and account id are required. Access token: '#{@access_token}'. Account ID: '#{@account_id}'.")
-      end
+      return unless @account_id.empty? || @access_token.empty?
```
Comment (bronzdoc) [inline lib/harvesting/client.rb:23]: I feel like the new code is harder to reason about.

I would prefer:
``` ruby
if @account_id.length.emtpy? || @access_token.empty?
  raise ArgumentError.new("Access token and account id are required. Access token: '#{@access_token}'. Account ID: '#{@account_id}'.")
end
```

What you think?

File: lib/harvesting/client.rb [hunk 2]
```diff
@@ -135,15 +136,11 @@ def get(path, opts = {})
     private
 
     def http_response(method, uri, opts = {})
-      response = nil
```
Comment (bronzdoc) [inline lib/harvesting/client.rb:138]: 👍

File: lib/harvesting/client.rb [hunk 3]
```diff
@@ -3,7 +3,6 @@
 require "json"
 
 module Harvesting
-
```
Comment (bronzdoc) [inline lib/harvesting/client.rb:6]: I'm in favor of removing this empty whitespace but just if we can enforce it using rubocop in our CI

File: lib/harvesting/client.rb [hunk 4]
```diff
@@ -135,15 +136,11 @@ def get(path, opts = {})
...
-      response = nil
-
       http = HTTP["User-Agent" => "Harvesting Ruby Gem",
                   "Authorization" => "Bearer #{@access_token}",
                   "Harvest-Account-ID" => @account_id]
       params = {}
-      if opts[:body]
-        params[:json] = opts[:body]
-      end
+      params[:json] = opts[:body] if opts[:body]
```
Comment (bronzdoc) [inline lib/harvesting/client.rb:143]: I'm in favor of this one liner but just if we can enforce it as part of our CI.

**Review by bronzdoc — COMMENTED** [review]
Thanks so much for taking the time to improve the Harvesting project @martinjaimem! 

I left some comments. I think we can add your style changes in another PR after we add some rubocop rules that enforce those style changes.

File: lib/harvesting/client.rb (hunk 1 above)
Comment (martinjaimem) [reply lib/harvesting/client.rb:23]: Yeah obviously!!!

we can configure it in the CI, I'd love to do that. I have more experience I Github Actions or CircleCI, If you want I can set one of them up to:

build, test, run rubocop

Comment (bronzdoc) [reply lib/harvesting/client.rb:23]: The Travis config for this project is quite simple so I don't have a problem to change to another CI. Can you open an issue for about this and explain what you have in mind?

**Pull request closed without merging**